openml-lightgbm-meta-module/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ The copyright of this file belongs to Feedzai. The file cannot be
+  ~ reproduced in whole or in part, stored in a retrieval system,
+  ~ transmitted in any form, or by any means electronic, mechanical,
+  ~ photocopying, or otherwise, without the prior permission of the owner.
+  ~
+  ~ (c) 2020 Feedzai, Strictly Confidential
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.feedzai</groupId>
+        <artifactId>openml-java</artifactId>
+        <version>0.0.0-SNAPSHOT</version>
+    </parent>
+    <packaging>pom</packaging>
+
+    <groupId>com.feedzai.openml.lightgbm.meta</groupId>
+    <artifactId>openml-lightgbm-meta-module</artifactId>
+    <description>OpenML LightGBM Machine Learning Model and Classifier</description>
+
+    <modules>
+        <!-- Builds LightGBM + lightgbmlib.jar -->
+        <module>lightgbm-builder</module>
+        <!-- The provider itself, which requires such artifacts -->
+        <module>openml-lightgbm</module>
+    </modules>
+
+    <properties>
+        <lightgbm.repo.url>https://github.com/microsoft/LightGBM</lightgbm.repo.url>
+        <lightgbm.version>fc79b366153e8aa5cf2996b4b753e28e5a75e878</lightgbm.version>
```

Comment:
I'm just raising some concerns.
But if @JPDSousa says that this is okay, then let's move on.
Just because we are in a democracy 😄

Reply:
This is not the commit of v2.3.1 @shenggwang, this time around is a commit from master because we can't add tags to their repo, and we need something else than their existing tags, but just in the previous release of our provider it was the tag v2.3.1.